dimagi/commcare-hq
# Refactor enterprise related logic to its own app

## Summary

Before adding more logic around Enterprise settings, we have decided to refactor it to its own app to simplify future changes related to enterprise accounts. This is simply a refactor of code that previously lived in the accounting app and now lives in the new enterprise app.

## Feature Flag


## Product Description


## Safety Assurance


### Automated test coverage


Adding a test for the `email enterprise report` task. 
### QA Plan


To ensure this refactor does not cause any regression, QA will run accounting related tests as well as enterprise settings related tests. 

[QA Ticket](https://dimagi-dev.atlassian.net/browse/QA-2191)
### Safety story

Tested enterprise pages locally. There is no change to behavior, just a refactor. If anything was wrong, this would only affect a small subset of users (enterprise admins). 
### Rollback instructions
